distributed-job-manager/src/main/java/ru/fix/distributed/job/manager/Worker.java

```diff
@@ -128,7 +128,7 @@ public void start() throws Exception {
             long workPoolCheckPeriod = v.getWorkPoolCheckPeriod();
             if (workPoolCheckPeriod != 0) {
                 workPoolReschedulableScheduler.schedule(
-                        DynamicProperty.of(() -> Schedule.withDelay(v.getWorkPoolCheckPeriod())),
+                        DynamicProperty.of(Schedule.withDelay(v.getWorkPoolCheckPeriod())),
```

Comment:
Could you please check: `DynamicProperty.delegated(()->Schedule.withDelay(v.getWorkPoolCheckPeriod()))`  - this way dynamic property will not be constant